CodeForAfricaLabs/CitizenReporter.Android
# Ft add flurry analytics

## PR Status
**READY**

## Story ID
https://trello.com/c/oendn2qz/45-analytics-android


## Description
A few sentences describing the overall goals or tasks of the pull request's commits.

## New Packages Added

com.flurry.android:analytics:7.0.0@aar

## Related PRs
NONE


## Todos
- [ ] Add analytics for google sign-up and sign-in

## Deploy Notes
* Create a new application at https://y.flurry.com
* Get app API key from flurry after creating a new application
* Replace your key with the key in CitizenReporterApplication

## Steps to Test or Reproduce
* Build the application using Android studio locally.
* Replace the API key with your own.
* Run the application on your phone or emulator
* Go to the flurry dashboard and see the tracked events

## Impacted Areas in Application
List general components of the application that this PR will affect:
* Sign up
* Login
* Storyboard activity
* Assignments detail activity
* Launcher activity


## Known Issues
* The events take some time to show in the flurry dashboard. They show in the logs though.
